cmd/buildah/images.go

```diff
@@ -142,10 +142,26 @@ func imagesCmd(c *cobra.Command, args []string, iopts *imageResults) error {
 		options.Filters = append(options.Filters, "intermediate=false")
 	}
 
-	images, err := runtime.ListImages(ctx, args, options)
+	imagesByNames, err := runtime.ListImagesByNames(args)
 	if err != nil {
 		return err
 	}
+	imagesByFilter, err := runtime.ListImages(ctx, options)
+	if err != nil {
+		return err
+	}
+
+	imagesByNameIDs := map[string]bool{}
```

Comment:
- If this is a set, it can be a `map[string]struct{}`.
- For a set of image IDs, the name seems misleading to me